src/util/vehicle.hpp

```diff
@@ -338,7 +338,7 @@ namespace big::vehicle
 			owned_mods[MOD_TIRESMOKE_COL_R] == 255 &&
 			owned_mods[MOD_TIRESMOKE_COL_G] == 255 &&
 			owned_mods[MOD_TIRESMOKE_COL_B] == 255
-		);
+			);
```

Comment:
Indent is still there

Reply:
removing it causes shit..

Reply:
Shit?

An indent is just the whitespace usually there are 2 ways they occur in code:
1. Using spaces `    `
2. Using tabs `	`

If you don't understand my remarks you can say so, or alternatively look it up on Google:
https://www.google.com/search?q=programming+indentation